editor/src/components/canvas/canvas-strategies/canvas-strategies.tsx

```diff
@@ -234,12 +235,41 @@ export function applyCanvasStrategy(
   return strategy.apply(canvasState, interactionSession, strategyState)
 }
 
+const useDelayedCurrentStrategy = () => {
+  const [currentStrategyValue, setCurrentStrategyValue] = React.useState<CanvasStrategyId | null>(
```

Comment:
Maybe slightly rename this, as `currentStrategyValue` seems a bit too close to `currentStrategy` to me.
```suggestion
  const [previousStrategy, setPreviousStrategy] = React.useState<CanvasStrategyId | null>(
```

Reply:
Maybe `previous` isn't a great prefix, perhaps `delayed`?

Reply:
thanks! updated the names in ab1b91ed7